docs/AboutUs.md

```diff
@@ -2,7 +2,7 @@
 
 Display |        Name        | Github Profile | Portfolio 
 --------|:------------------:|:--------------:|:---------:
-![](https://via.placeholder.com/100.png?text=Photo) | Cheng Jia Wei Andy | [Github](https://github.com/averageandyyy) | [Portfolio](docs/team/johndoe.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Cheng Jia Wei Andy | [Github](https://github.com/averageandyyy) | [Portfolio](docs/team/chengjiaweiandy.md)
```

Comment:
Should you leave the `docs/` part of the directory name out? Since it is in the `docs` directory already.